curation_concerns-models/app/models/concerns/curation_concerns/upload_set_behavior.rb

```diff
@@ -7,7 +7,6 @@ module UploadSetBehavior
     included do
       has_many :file_sets, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf
 
-      property :creator, predicate: ::RDF::DC.creator
```

Comment:
apply_depositor_metadata persists associated web acls into the repo. This will slow things down. Are you sure we want that?